runtime/src/main/java/org/corfudb/protocols/wireprotocol/LogData.java

```diff
@@ -74,18 +79,26 @@ public LogData(DataType type) {
         this.metadataMap = new EnumMap<>(IMetadata.LogUnitMetadataType.class);
     }
 
-    public LogData(DataType type, ByteBuf buf) {
+    public LogData(final DataType type, final ByteBuf buf) {
```

Comment:
ah, I see. I wanted this change to be small, changing the method signature would have changed everything else that created a LogData. I can take care of that too.